locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -1522,8 +1530,9 @@ msgid "Resolve"
 msgstr "Résoudre"
 
 #: bookwyrm/templates/moderation/reports.html:6
-#, python-format
-msgid "Reports: %(server_name)s"
+#, fuzzy, python-format
+#| msgid "Reports: %(server_name)s"
+msgid "Reports: %(instance_name)s"
 msgstr "Signalements : %(server_name)s"
```

Comment:
```suggestion
msgid "Reports: %(instance_name)s"
msgstr "Signalements : %(instance_name)s"
```

Reply:
oh I need to remember to fix this variable name on the other locales! thank you